workers/datasets_based/tests/workers/test__datasets_based_worker.py

```diff
@@ -31,7 +34,7 @@ def compute(self) -> Mapping[str, Any]:
         if self.config == "raise":
             raise ValueError("This is a test")
         else:
-            return {}
+            return {"col1": ["a" * 10 for _ in range(200)]}
```

Comment:
maybe something simpler to understand, and that should also be greater than the max size

```suggestion
            return {"col1": "a" * 200}
```